osg-comanage-project-usermap.py

```diff
@@ -146,6 +149,35 @@ def get_osguser_groups(filter_group_name=None):
              for pid, gids in pid_gids.items() }
 
 
+def parse_inputfile(inputfile):
+    user_groupmap = dict()
+    with open(inputfile) as file:
+        for line in file:
+            split_line = line.split()
```

Comment:
```suggestion
            split_line = line.strip().split(maxsplit=2)
```

I think that you're mainly looking to get rid of the trailing newline here?